js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -164,7 +182,14 @@ usermodel.clientRoleMapping.clientId.label=Client-ID
 manage=Verwalten
 syncModeHelp=Standardsyncmodus für alle Mapper. Mögliche Werte sind\: 'Legacy' um das alte Verhalten beizubehalten, 'Importieren' um den Nutzer einmalig zu importieren, 'Erzwingen' um den Nutzer immer zu importieren.
 temporaryPassword=Temporär
-userManagedAccess=Wenn aktiviert, können Benutzer ihre Ressourcen und Berechtigungen über die Account Management UI verwalten.
+userManagedAccess=User Managed Access
```

Comment:
benutzerverwalteter Zugriff

Reply:
Okay I go with "User-Managed Access (UMA)" as it is the official term.